### DISCUSSION TITLE
Suggestion: Microsoft Store version

### DISCUSSION THREAD

**Discussion by myokeeh** [pawelsalawa/sqlitestudio#3553] [Q&A]
Have you considered submitting the app to the Microsoft Store even for a price or a donation via paid add-on (consumable)?

**Comment by pawelsalawa** [comment]
I could consider it, but it takes time and effort to prepare & release it there, while I barely have time to work on actual code these days, so I guess right now it's more important to work on the code than dealing with MS Store.

I'll leave it here as idea for future.
